tests/route/conftest.py

```diff
@@ -1,3 +1,5 @@
+import pytest
```

Comment:
The new `test_router_flap.py` is included in this change. Can you create a separate PR for it.